eq-author/src/App/page/Design/QuestionPageEditor/AnswerTypeSelector/index.js

```diff
@@ -75,13 +75,21 @@ class AnswerTypeSelector extends React.Component {
     this.grid = grid;
   };
 
+ 
+
   render() {
     const errorValidationMsg = this.props.getValidationError({
       field: "answers",
       message: QUESTION_ANSWER_NOT_SELECTED,
     });
     let hasDateRange = false;
     let hasOtherAnswerType = false;
+    
+    let mutuallyExclusive = false;
+    // TODO-----------------add this once options from backend is merged in------------------;;;
+    // const answers = Array.from(this.props.page.answers);
+    // mutuallyExclusive = { ...answers?.options?.some(({ mutuallyExclusive }) => mutuallyExclusive === true)};
+
```

Comment:
Does this need to be done now the backend is merged in?

Reply:
yes - thanks MAtt - nowupdated and test added